src/app/content/components/Sidebar/styled/index.tsx

```diff
@@ -11,6 +9,8 @@ import ContentLinkComponent from '../../ContentLink';
 
 export * from './wrapper';
 
+export {Summary, ExpandIcon, CollapseIcon} from '../../../../components/Details';
```

Comment:
Just In Case ™️ we decide to add some style overrides to those then we don't have to change the imports in the sidebar

if it bothers you we can remove, not a big deal

Reply:
I just didn't expect to see it, that's all.